scripts/install-worker.sh

```diff
@@ -473,7 +474,7 @@ fi
 ### SSM Agent ##################################################################
 ################################################################################
 
-sudo yum install -y amazon-ssm-agent
+sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/${SSM_AGENT_VERSION}/linux_amd64/amazon-ssm-agent.rpm
```

Comment:
I think you'll want to look at how other artifacts are pulled from region specific buckets https://github.com/awslabs/amazon-eks-ami/blob/55cbc55f69a029dc821d16081a7c75c206a5d1cd/scripts/install-worker.sh#L245-L255

Reply:
I've just added a variable to configure the region.
Note that according [the documentation](https://docs.aws.amazon.com/systems-manager/latest/userguide/agent-install-al2.html#custom-url-al2), the supported regions are the same as those defined on [Systems Manager service endpoints](https://docs.aws.amazon.com/general/latest/gr/ssm.html#ssm_region).

Reply:
you should be able to re-use the already existing `$BINARY_BUCKET_REGION` and `$S3_DOMAIN` variables instead of adding new. and the domain will need to be updated as well

Reply:
Are you sure about this?
Looking at the documentation I put in this comment thread, the regions cn-north-1 / cn-northwest-1 / us-iso-east-1 / us-iso-west-1 / us-isob-east-1 are not supported.
That's why I chose to use a dedicated `SSM_AGENT_RPM_BUCKET_REGION` variable.

Reply:
@vincentmrg SSM is definitely supported in those partitions, the docs are just elsewhere. E.g. https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-Beijing.html

Reply:
Thanks for the explanation, I'm not familiar with these particular regions, I've just pushed the requested changes.

Reply:
Looks correct, but @cartermckinnon will have the final say